Library/BillingInfo.cs

```diff
@@ -15,7 +15,12 @@ public enum CreditCardType : short
             MasterCard,
             AmericanExpress,
             Discover,
-            JCB
+            JCB,
+            Danokrt,
+            Maestro,
+            Forbrugsforeningen,
+            Laser,
+            Unknown

```

Comment:
A couple of these values actually have spaces in them. I left them out. I wonder if we should change this to a string instead of an enum.